frontend/src/routes/dashboard/[groupId]/+page.svelte

```diff
@@ -67,15 +70,27 @@
 		} else {
 			console.error('Invalid start or end date');
 		}
+
+		const pollingInterval = setInterval(async () => {
+			await getGroup(groupId);
+			await calculateCommonAvailability();
+		}, 1000);
```

Comment:
Have you considered reactive variables as an alternative to polling the backend every second?
```js
// +page.svelte
$: group = <already exists>;
$: calculateCommonAvailability(group);
```
Svelte will auto-re-run the `calculateCommonAvailability` function any time `group` changes.